addons/udes_advanced_packaging/views/stock_picking_type.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_type_form_inherit" model="ir.ui.view">
+        <field name="name">stock.picking.type.form.inherit</field>
+        <field name="model">stock.picking.type</field>
+        <field name="inherit_id" ref="stock.view_picking_type_form"/>
+        <field name="arch" type="xml">
+
+            <!-- Add udes fields -->
+            <xpath expr="//form/sheet/group[last()]" position="after">
+                <group class="udes_wide_list" string="UDES Config" groups='base.group_no_one'>
```

Comment:
CSS class `udes_wide_list` doesn't exist, we don't need it here though so can get rid of this class attribute.

Also we should set the string to "Packagisation" or "Advanced Packaging". In v11 we've got one group with a million config options, but now we can split out the config options into their own groups for each module. :)

Reply:
Fixed in latest commit

Reply:
Small nit on that, could we have "Advanced Packaging" instead of Packing for the group string?

Reply:
Fixed